experimental/LieAlgebras/src/AbstractLieAlgebra.jl

```diff
@@ -114,6 +114,20 @@ end
 #
 ###############################################################################
 
+@doc raw"""
+    lie_algebra(R::Ring, struct_consts::Matrix{SRow{C}}, s::Vector{<:VarName}; cached::Bool, check::Bool) -> AbstractLieAlgebra{elem_type(R)}
+
+Construct the Lie algebra over the ring `R` with structure constants given by
+`struct_consts` and with basis element names `s`.
+
+* `struct_consts`: The entry with indices `[i,j][k]` is the scalar $a_{i,j}^k$
```

Comment:
Personally I'd rename `struct_consts` to something short, e.g. `SC`, and then write more like the following (but feel free to disagree and ignore this). Also, while I know that `a_{i,j}^k` is semi-standard, it is still easy to get confused and think the `^x` refers to exponentiation. So here is what I'd write:
```suggestion
The Lie bracket on the newly constructed Lie algebra `L` is determined by the structure
constants in `SC` as follows: let `x_i` denote the `i`-th standard basis vector
of `L`. Then the entry `SC[i,j][k]` is a scalar $a_{i,j,k}$
```

Reply:
I personally think that `struct_consts` is better following the principle of self-commenting code. Some of your wording has been adapted in e19efeaca58f9bdf1ffa03cff8ed36f8c5cde75d.